### PR TITLE
feat(flags): add relative timestamp to flag tooltip

### PR DESCRIPTION
**Pull request by michellewzhang**
SCR-20241021-laxc

### CODE REVIEW

File: static/app/views/issueDetails/streamline/flagSeries.tsx [hunk 1]
```diff
@@ -110,6 +117,8 @@ export default function useFlagSeries({query = {}}: {query?: Record<string, any>
           '</div>',
           '<div class="tooltip-footer">',
           time,
+          event.dateCreated &&
```
Comment (aliu39) [inline static/app/views/issueDetails/streamline/flagSeries.tsx:120]: Does this && append the strings?

Comment (michellewzhang) [reply static/app/views/issueDetails/streamline/flagSeries.tsx:120]: it checks that `dateCreated` exists

**Review by c298lee — APPROVED** [review]

File: static/app/views/issueDetails/streamline/flagSeries.tsx (hunk 1 above)
Comment (aliu39) [reply static/app/views/issueDetails/streamline/flagSeries.tsx:120]: Oh so is the `time` on line 119 where we pass the absolute date? Could we get the date from the same source, `data.xAxis`?

Comment (c298lee) [reply static/app/views/issueDetails/streamline/flagSeries.tsx:120]: i think `time` is when the flag was created whereas `event.dateCreated` is when the event was created

Comment (michellewzhang) [reply static/app/views/issueDetails/streamline/flagSeries.tsx:120]: yup, @c298lee 's assumption is correct

Comment (michellewzhang) [reply static/app/views/issueDetails/streamline/flagSeries.tsx:120]: we want to show the flag's change/creation/deletion time in relation to the event creation date

Comment (aliu39) [reply static/app/views/issueDetails/streamline/flagSeries.tsx:120]: Oh I get it now! Thanks both of you

**Review by aliu39 — APPROVED** [review]

**Pull request merged**